.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules
 coverage
 coverage.json
 typechain
+deployments
```

Comment:
@jdaly13 @gnomadic this is the folder where you'll see all the contract addys.

My thought is that we could maybe get a bit more specific on this and just ignore test files, while adding mainnet deployments information.

Reply:
addys? Yeah I would say ignore subfolders but keep deployments?